### PR TITLE
Correct affected versions for nayzo/NzoUrlEncryptorBundle

### PR DESCRIPTION
**Pull request by nayzo**

### CODE REVIEW

File: nzo/url-encryptor-bundle/2020-05-03.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ cve:       ~
 branches:
     "5.x":
         time:     2020-05-03 16:54:00
-        versions: [<5.0.1]
+        versions: [<4.3.2, 5.0.0]
```
Comment (fabpot) [inline nzo/url-encryptor-bundle/2020-05-03.yaml:7]: Can you create two branches? One for 4.x and one for 5.x?

Comment (nayzo) [reply nzo/url-encryptor-bundle/2020-05-03.yaml]: Done.

File: nzo/url-encryptor-bundle/2020-05-03.yaml [hunk 2]
```diff
@@ -2,7 +2,10 @@ title:     Insecure default secret key and IV allowing anyone to decrypt values
...
 branches:
-    "5.x":
-        time:     2020-05-03 16:54:00
-        versions: [<5.0.1]
+    5.x:
+        time:     2020-05-04 14:50:00
+        versions: [5.0.0]
+    4.x:
+        time:     2020-05-04 14:50:00
+        versions: [<4.3.2]
```
Comment (fabpot) [inline nzo/url-encryptor-bundle/2020-05-03.yaml:10]: `['>=4.0.0', '<4.3.2']`

File: nzo/url-encryptor-bundle/2020-05-03.yaml [hunk 3]
```diff
@@ -2,7 +2,10 @@ title:     Insecure default secret key and IV allowing anyone to decrypt values
 link:      https://github.com/nayzo/NzoUrlEncryptorBundle/issues/41
 cve:       ~
 branches:
-    "5.x":
-        time:     2020-05-03 16:54:00
-        versions: [<5.0.1]
+    5.x:
+        time:     2020-05-04 14:50:00
+        versions: [5.0.0]
```
Comment (fabpot) [inline nzo/url-encryptor-bundle/2020-05-03.yaml:7]: `['>=5.0.0', '<5.0.1']`

File: nzo/url-encryptor-bundle/2020-05-03.yaml (hunk 2 above)
Comment (nayzo) [reply nzo/url-encryptor-bundle/2020-05-03.yaml]: Done.

File: nzo/url-encryptor-bundle/2020-05-03.yaml (hunk 3 above)
Comment (nayzo) [reply nzo/url-encryptor-bundle/2020-05-03.yaml]: Done.

**Pull request merged**